src/yowsup_if.js

```diff
@@ -96,9 +96,7 @@ CoSeMe.namespace('yowsup.readerThread', (function() {
     get: function(iqType, idx, node) {
       var childNode = node.getChild(0);
       if (node.getAttributeValue('xmlns') === 'urn:xmpp:ping') {
-        if (_autoPong) {
```

Comment:
so it only uses a single callback mechanism (_signalInterface) instead of 2 completely separate (_signalInterface and the _onPing setter/getter functions)